.pre-commit-config.yaml

```diff
@@ -19,6 +19,8 @@ repos:
   rev: v2.3.0
   hooks:
   - id: codespell
+    additional_dependencies:
```

Comment:
Just out of curiosity. Is this respecting the config you declared in the ``pyproject.toml`` file?

Reply:
yes!